app/views/decidim/vocdoni/admin/answers/index.html.erb

```diff
@@ -1,12 +1,11 @@
 <% add_decidim_page_title(t(".title")) %>
```

Comment:
Let's introduce a break-line (as it was before) so it easy to have the meta and body blocks visually separated. 

```suggestion
<% add_decidim_page_title(t(".title")) %>

```